api/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Mvc;
+
+[ApiController]
+[Route("[controller]")]
+public class WeatherForecastController : ControllerBase
+{
+    private static readonly string[] Summaries = new[]
+    {
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+    };
+
+    [HttpGet]
+    public IEnumerable<WeatherForecast> Get()
```

Comment:
Every public method should have documentation comment (see https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments)